web/src/compose_recipient.js

```diff
@@ -19,9 +21,17 @@ import * as stream_data from "./stream_data";
 import * as ui_util from "./ui_util";
 import * as util from "./util";
 
-export let compose_recipient_widget;
+// selected_recipient_id is used store which recipient is selected:
```

Comment:
"selected_recipient_id stores which recipient is selected:"

Reply:
Changed this.

Reply:
Thanks, my bad.